### PR TITLE
worked on otp and singup screen UI and functionality

### PR DESCRIPTION
**Pull request by DeepakThakur27**
Hi, 
Navin check this changes I have created two screen and also worked on functionality

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const Home = () => {
+    return (
+        <>
+            <h1>WelCome to Home page</h1>
```
Comment (navinwadbude) [inline src/Home.js]: why are add unnecessary tag

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+      <Router>
+
+        <Routes>
+          <Route exact path='/' element={< UserForm />}></Route>
+          <Route exact path='/otp' element={< Otp />}></Route>
+          <Route exact path='/home' element={< Home />}></Route>
+        </Routes>
+
+      </Router>
+    </>
```
Comment (navinwadbude) [inline src/App.js]: The same extra tag add. remove that

File: src/Otp.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import OtpInput from "react-otp-input";
+
+import './style.css';
+
+export const Otp = () => {
+    const location = useLocation();
+    const navigate = useNavigate();
+    // let formOtp = location?.state;
```
Comment (navinwadbude) [inline src/Otp.js]: remove commented code

File: src/Otp.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        <>
+            <h4 style={{ justifyContent: 'center', alignContent: 'center', textAlign: 'center' }}>OTP Screen</h4>
+            <div className="main">
+                <div className="otpContainer">
+                    <OtpInput
+                        value={otp}
+                        onChange={setOtp}
+                        numInputs={4}
+                        renderSeparator={<span> - </span>}
+                        renderInput={(props) => <input {...props} style={{ borderRadius: '5px', width: '35px', height: '35px', textAlign: 'center' }} />}
```
Comment (navinwadbude) [inline src/Otp.js]: don't pass inline css. create new file and all css inside that

File: src/UserForm.js [hunk 5]
```diff
@@ -23,10 +24,10 @@ export const UserForm = () => {
 
     const onSubmit = data => {
         const otp = Math.trunc(Math.random() * 10001);
-        console.log(otp);
```
Comment (navinwadbude) [inline src/UserForm.js:26]: remove console.log

File: src/Otp.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+            clearInterval(setTime)
+        }
+
+    };
+
+    const resend = () => {
+
+        const NewOtp = Math.trunc(Math.random() * 10001);
+        setGenOtp(NewOtp)
+        console.log('resend', genOtp);
```
Comment (navinwadbude) [inline src/Otp.js:33]: remove console.log

File: src/App.js [hunk 7]
```diff
@@ -1,9 +1,27 @@
...
+    // <UserForm />
+    // <Otp />
```
Comment (navinwadbude) [inline src/App.js]: remove commented code

File: src/Home.js (hunk 1 above)
Comment (DeepakThakur27) [reply src/Home.js]: done

File: src/App.js (hunk 2 above)
Comment (DeepakThakur27) [reply src/App.js]: done

File: src/Otp.js (hunk 3 above)
Comment (DeepakThakur27) [reply src/Otp.js]: all commented has been removed

File: src/UserForm.js (hunk 5 above)
Comment (DeepakThakur27) [reply src/UserForm.js:26]: It is used for showing the otp for user

File: src/Otp.js (hunk 6 above)
Comment (DeepakThakur27) [reply src/Otp.js:33]: It is used for showing the resend otp for user

File: src/App.js (hunk 7 above)
Comment (DeepakThakur27) [reply src/App.js]: removed all commented

File: src/Otp.js (hunk 4 above)
Comment (DeepakThakur27) [reply src/Otp.js:70]: Inline css has used due to library structure

**Pull request merged**